DemoServer/AdditionalLanguages/nodejs/demo/queries/queryOverview/metadata.json

```diff
@@ -0,0 +1,209 @@
+{
+    "Slug": "query-overview",
+    "SourceFileName": "index.js",
+    "Title": "Query Overview",
+    "NonInteractive":  true,
+    "DescriptionLines": [
+        "RavenDB has rich querying and indexing support.",
+        "",
+        "**Query the database to find documents that match a particular predicate**.",
+        "The server will return only the matching documents.",
+        "",
+        "The returned documents are loaded and tracked by the Session (except for projections).",
+        "",
+        "From the session, query the database by using any of:",
```

Comment:
I'd add parens to each of those below.

Reply:
Imho - 
This is just a short summary area - not exact code sample - 
It is there just to **get the reader focused** on the fact that there are '3 ways to go' -
giving a beginner a high concept of what is available -

For that same reason - the last option has `(using RQL) `

To me - adding 'parens' is too much for the eye in this short summary
(it is like this in all languages)
=> Pls let me know if you think otherwise - thanks

Reply:
Well, I like the method names with parentheses rather than without. That's the standard way of indication the name is a method / function. It's your call, that's just a suggestion.